src/templates/finops-hub/main.bicep

```diff
@@ -46,3 +46,6 @@ output location string = location
 
 @description('The resource ID of the deployed storage account.')
 output storageAccountId string = hub.outputs.storageAccountId
+
+@description('The Resource ID of the Data factory.')
+output dataFactoryId string = hub.outputs.dataFactoryId
```

Comment:
Same as in hub.bicep. Either remove or add to docs.

Reply:
removed